wazo_acceptance/steps/user.py

```diff
@@ -106,6 +106,10 @@ def given_there_are_telephony_users_with_infos(context):
             agent = context.helpers.agent.create({'number': body['agent_number']})
             context.confd_client.users(confd_user).add_agent(agent)
 
+        if body.get('schedule'):
```

Comment:
Since you already have the `Given "Clark Kent" has schedule "closed_in_open"` step, I will not allow to have schedule in the user create step